components/org.wso2.carbon.identity.oauth.endpoint/src/test/resources/repository/deployment/server/webapps/authenticationendpoint/oauth_response.jsp

```diff
@@ -0,0 +1,24 @@
+<!--
+~    Copyright (c) 2022, WSO2 Inc. (http://www.wso2.com). All Rights Reserved.
+~
+~    This software is the property of WSO2 Inc. and its suppliers, if any.
+~    Dissemination of any information or reproduction of any material contained
+~    herein in any form is strictly forbidden, unless permitted by WSO2 expressly.
+~    You may not alter or remove any copyright or other notice from copies of this content."
+-->
+
+<html>
+<head>
+    <title>Asgardeo by WSO2</title>
```

Comment:
We shouldn't have solution details to the public repo commits

Reply:
Fixed with fcba3c712827c2b8920198cb6431095679161835